src/orchestrator-process/src/lib.rs

```diff
@@ -145,7 +145,10 @@ impl NamespacedOrchestrator for NamespacedProcessOrchestrator {
         &self,
         _id: &str,
     ) -> Result<Vec<ServiceProcessMetrics>, anyhow::Error> {
-        anyhow::bail!("metrics are not supported on the process orchestrator");
+        // Metrics are not currently supported on the process orchestrator
+        loop {
+            tokio::time::sleep(Duration::MAX).await;
+        }
```

Comment:
In other instances we use `future::pending().await;` (https://docs.rs/futures/latest/futures/future/fn.pending.html). I guess that would be the most idiomatic way to implement this.

Reply:
Thanks. I was looking for something like that, but my Google queries came up empty...

(well, "tokio park" was one thing I tried googling, and that showed me some nice green spaces in Japan ...)